site/package.json

```diff
@@ -25,7 +27,7 @@
     "gulp-htmlmin": "^5.0.1",
     "gulp-minify": "^3.1.0",
     "gulp-pug": "^4.0.1",
-    "gulp-responsive": "^2.11.0",
+    "gulp-responsive": "mahnunchik/gulp-responsive#greenkeeper/sharp-0.21.0",
```

Comment:
Can we lock this down to a commit rather than a branch?

Reply:
sure